tests/lib/state_test.py

```diff
@@ -157,6 +158,32 @@ def test_sort_multiple_ip(self):
 
         desired_state.verify_interfaces(current_state)
 
+    def test_sort_ovs_lag_slaves(self):
+        desired_state = self._base_state
+        current_state = self._base_state
+        slaves = [
+            {OVSBridge.Port.LinkAggregation.Slave.NAME: "slave1"},
+            {OVSBridge.Port.LinkAggregation.Slave.NAME: "slave2"},
+        ]
+        d_ovs = {
+            OVSBridge.PORT_SUBTREE: [
+                {
+                    OVSBridge.Port.NAME: "boo",
+                    OVSBridge.Port.LINK_AGGREGATION_SUBTREE: {
+                        OVSBridge.Port.LinkAggregation.SLAVES_SUBTREE: slaves
+                    },
+                }
+            ]
+        }
+        c_ovs = copy.deepcopy(d_ovs)
+        c_port0 = c_ovs[OVSBridge.PORT_SUBTREE][0]
+        c_lag0 = c_port0[OVSBridge.Port.LINK_AGGREGATION_SUBTREE]
+        c_lag0[OVSBridge.Port.LinkAggregation.SLAVES_SUBTREE].reverse()
+        desired_state.interfaces["foo-name"][OVSBridge.CONFIG_SUBTREE] = d_ovs
+        current_state.interfaces["foo-name"][OVSBridge.CONFIG_SUBTREE] = c_ovs
```

Comment:
You did not do deep copy while initializing desired and current states. So here you update both variables with d_ and c_. Is that intended?

Reply:
‘self._base_state’ returns a new dict each time it is called. Is this what you meant?

Reply:
I see. All good then